[2023-11-05T15:54:38.064-0800] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: source_extracting_dag.unzip_sales_data_task manual__2023-11-05T23:54:19.599416+00:00 [queued]>
[2023-11-05T15:54:38.070-0800] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: source_extracting_dag.unzip_sales_data_task manual__2023-11-05T23:54:19.599416+00:00 [queued]>
[2023-11-05T15:54:38.070-0800] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T15:54:38.070-0800] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-05T15:54:38.070-0800] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T15:54:38.078-0800] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): unzip_sales_data_task> on 2023-11-05 23:54:19.599416+00:00
[2023-11-05T15:54:38.083-0800] {standard_task_runner.py:55} INFO - Started process 90112 to run task
[2023-11-05T15:54:38.092-0800] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'source_extracting_dag', 'unzip_sales_data_task', 'manual__2023-11-05T23:54:19.599416+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/source_extract_dag.py', '--cfg-path', '/var/folders/jh/g6c8fm7j3x148qd22mrgmrsw0000gn/T/tmpk1khpi6r']
[2023-11-05T15:54:38.096-0800] {standard_task_runner.py:83} INFO - Job 106: Subtask unzip_sales_data_task
[2023-11-05T15:54:38.135-0800] {task_command.py:388} INFO - Running <TaskInstance: source_extracting_dag.unzip_sales_data_task manual__2023-11-05T23:54:19.599416+00:00 [running]> on host anishs-air.lan
[2023-11-05T15:54:38.156-0800] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=source_extracting_dag
AIRFLOW_CTX_TASK_ID=unzip_sales_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-11-05T23:54:19.599416+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-05T23:54:19.599416+00:00
[2023-11-05T15:54:38.156-0800] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/jh/g6c8fm7j3x148qd22mrgmrsw0000gn/T
[2023-11-05T15:54:38.157-0800] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'unzip /Users/anishmore/airflow/data/snowflake-work/snowpark-example/end2end-sample-data/3-region-sales-data.zip -d /Users/anishmore/airflow/sources']
[2023-11-05T15:54:38.160-0800] {subprocess.py:86} INFO - Output:
[2023-11-05T15:54:38.165-0800] {subprocess.py:93} INFO - unzip:  cannot find or open /Users/anishmore/airflow/data/snowflake-work/snowpark-example/end2end-sample-data/3-region-sales-data.zip, /Users/anishmore/airflow/data/snowflake-work/snowpark-example/end2end-sample-data/3-region-sales-data.zip.zip or /Users/anishmore/airflow/data/snowflake-work/snowpark-example/end2end-sample-data/3-region-sales-data.zip.ZIP.
[2023-11-05T15:54:38.166-0800] {subprocess.py:97} INFO - Command exited with return code 9
[2023-11-05T15:54:38.169-0800] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 9.
[2023-11-05T15:54:38.171-0800] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=source_extracting_dag, task_id=unzip_sales_data_task, execution_date=20231105T235419, start_date=20231105T235438, end_date=20231105T235438
[2023-11-05T15:54:38.175-0800] {standard_task_runner.py:100} ERROR - Failed to execute job 106 for task unzip_sales_data_task (Bash command failed. The command returned a non-zero exit code 9.; 90112)
[2023-11-05T15:54:38.192-0800] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-11-05T15:54:38.200-0800] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
